draft-tuexen-tsvwg-dtls-chunk-key-management.xml

```diff
@@ -110,15 +110,62 @@ they appear in all capitals, as shown here.</t>
 
 <section>
 <name>Architecture</name>
-<t>TBD</t>
+<t>This document describes how Datagram Transport Layer Security (DTLS) 1.3 is
+used to establish keys for securing SCTP using the DTLS Chunk. This approach
+combines the performance and encryption flexibility of DTLS Chunks with the
+integrated key management capabilities of DTLS. The resulting solution
+can be viewed from two perspectives: as a new key management method for
+SCTP, and as an SCTP-specific encryption layer integrated with DTLS.
+</t>
+
+<t>The key characteristics of the solution are as follows:</t>
+<ul>
+<li><t>Application data is protected using DTLS Chunks.</t></li>
+<li><t>The DTLS handshake is used to establish keying material, algorithms,
+and parameters for use with DTLS Chunks.</t></li>
+<li><t>DTLS and relevant extensions are used to negotiate cryptographic
+algorithms and parameters, perform extended key updates, enable larger
+record sizes, and support post-handshake authentication.</t>
+<li><t>All other DTLS record-layer content types are protected using
+standard DTLS record framing.</t></li>
+</ul>
+
+<t>Application data is never transmitted in DTLS record-layer application_data
+records. Instead, application data is passed to the DTLS stack, which forwards
+it to the Record Protection Operator. The operator encapsulates the data into
+DTLS Chunks, applying appropriate protection.</t></li>
+
+<t>Any data that is not part of a DTLS Chunk MUST use standard DTLS record-layer
+framing and MUST be sent in separate datagrams.</t>
+
+<t>The figure below illustrates the architecture, highlighting the
+role of the upper-layer protocol (ULP), which acts as the consumer of
+SCTP's transport services. The ULP may interface directly with the SCTP
+library or operate through the DTLS 1.3 stack. This specification builds
+upon DTLS 1.3 by introducing additional extensions to support enhanced
+functionality, including post-handshake authentication as described in
+<xref target="RFC9261"/> and the extended key update mechanism defined
+in <xref target="I-D.ietf-tls-extended-key-update"/>.</t>
+
+<t>Following the initial SCTP association setup, a DTLS 1.3 handshake
+is performed to mutually authenticate the endpoints and to derive keying
+material for the DTLS Chunk Protection Operator. The TLS exporter, as
+defined in Section 7.5 of <xref target="RFC8446"/>, is used to derive
+this keying material. It leverages the same cryptographic algorithms
+that were negotiated during the DTLS handshake for use with the DTLS
+Record Layer, thereby eliminating the need for separate algorithm
+negotiation for the DTLS Chunk. However, this approach requires that
+only algorithm suites compatible with both DTLS 1.3 and DTLS Chunks be
+configured and supported.</t>
+     
 <figure>
 <name>Architecture</name>
 <artwork align="left">
 +---------------+ +-------------------------------+
-|      ULP      | | Post Handshake Authentication |
-|               | |        using RFC 9261         |
```

Comment:
My understanding is that the Extended Key Update runs within the DTLS layer, not on top of it. So Why are you drawing it on top of DTLS?